### ISSUE TITLE
documentation is not translated 

### ISSUE THREAD

**Issue by icq4ever** [openframeworks/ofSite#524]
I uploaded korean translation of ofGraphics. but it seems not use *.ko,markdown but .markdown.
check [ko.markdown](https://github.com/openframeworks/ofSite/blob/master/documentation/graphics/introduction.ko.markdown) and [rendered pages](http://openframeworks.cc/ko/documentation/graphics/)

**Comment by arturoc** [comment]
Yes the docs still don't have translation. i didn't want to add extra complexity unless there were interest in actually translating them but if you are going to do it i can look into it.

**Comment by icq4ever** [comment]
I see~ sure I'm interest in.. but at the moment not ready for start to hard working.
when we're ready for that, I'll send DM :) thanks!

**Comment by arturoc** [comment]
great, i'll leave this open, there's also the book which is probably easier to translate and there's still some effort going on to translate it into japanese so perhaps it's better to start with that.

**Comment by icq4ever** [comment]
did you mean ofBook project ?

**Comment by arturoc** [comment]
yeah that's it, there's even some chapters that are already translated since they were tutorials before
